src/components/Comment/Comment.tsx

```diff
@@ -173,8 +177,10 @@ const flexRowStyles = css`
   flex-direction: row;
 `;
 
-const removePaddingLeft = css`
-  padding-left: 0px;
+const removeButtonPaddingLeft = css`
+  button {
+    padding-left: 0px;
+  }
```

Comment:
This rule conflicts with how padding is set in the `commentControlsButtonWrapper ` css above

Reply:
this raises an interesting issue, we need to remove padding left from only the 1st button and that button is not necessarily the reply button.

Reply:
I think a good pattern in these cases is to make the components margin/padding agnostic and to extract spacing into context specific components. It also solves the only pad when not first item problem.

An example of this is in `FirstCommentWelcome`:

```
        <Row>
          <Button size="small" onClick={() => submitForm(userName)}>
            Post your comment
          </Button>
          <div
            className={css`
              width: ${space[3]}px;
            `}
          ></div>
          <Button size="small" priority="tertiary" onClick={cancelSubmit}>
            Cancel
          </Button>
        </Row>
```

Reply:
Like this, if we extracted the Button logic into a separate component it would read as:

```
        <Row>
          <ButtonLink size="small" onClick={() => doStuff()}>
            Reply
          </ButtonLink>
          <div
            className={css`
              width: ${space[3]}px;
            `}
          ></div>
          <ButtonLink size="small"onClick={otherStuff()}>
            Share
          </ButtonLink>
        </Row>
```

You could maybe (🤷‍♂) also abstract the spacing css to give:

```
        <Row>
          <ButtonLink size="small" onClick={() => doStuff()}>
            Reply
          </ButtonLink>
          <Space amount={3} />
          <ButtonLink size="small"onClick={otherStuff()}>
            Share
          </ButtonLink>
        </Row>
```